docs/tutorials/using-data-sources.md

```diff
@@ -9,28 +9,92 @@ import Tabs from '@theme/Tabs';
+
+In the configuration for a market, a data source specification field dictates which data feeds it is interested in. In effect, it works as a filter. This specification means that the creator of an instrument for a market will choose in advance a price source, and which data fields the market requires to settle and/or terminate.
 
 :::note Read more: 
-[Market governance concepts:](../concepts/governance.md#market-governance)
-[Tutorial - proposing a market:](./proposals/new-market-proposal.md)
+* [Market governance concepts](../concepts/governance.md#market-governance)
+* [Tutorial - proposing a market](./proposals/new-market-proposal.md)
+:::
+
+## Submitting data
+Markets that use Ethereum data sources don't need anyone to submit data. 
+
+For markets using other data source types, any Vega keypair can submit data. When it's time for a futures market using a non-Ethereum data source to settle, someone needs to submit the data that matches the data source spec defined in the market.
+
+## Ethereum oracles
+Since version 0.73.0, settlement data can be sourced from Ethereum smart contracts. The following spec would read from the Ethereum contract at `0x1b4...e43` every 30 seconds, and fetch the Bitcoin price value from the returned object:
+
+```javascript
+"dataSourceSpecForSettlementData": {
+    "external": {
+        "ethOracle": {
+            "address": "0x1b44F3514812d835EB1BDB0acB33d3fA3351Ee43",
+            "abi": "[{\"inputs\":[],\"name\":\"latestAnswer\",\"outputs\":[{\"internalType\":\"int256\",\"name\":\"\",\"type\":\"int256\"}],\"stateMutability\":\"view\",\"type\":\"function\"}]",
+            "method": "latestAnswer",
+            "normalisers": [{
+                  "name": "btc.price",
+                  "expression": "$[0]"
+              }],
+            "requiredConfirmations": 3,
+            "trigger": {
+                "timeTrigger": {
+                    "every": 30
+                }
+            },
+            "filters": [{
+                    "key": {
+                        "name": "btc.price",
+                        "type": "TYPE_INTEGER",
+                        "numberDecimalPlaces": 8
+                    },
+                    "conditions": [
+                        {
+                            "operator": "OPERATOR_GREATER_THAN_OR_EQUAL",
+                            "value": "0"
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+}
+```
+
+This will cause the network validators to read the result from the specified smart contract and submit the result to Vega. When the data is verified by enough validators, this price is accepted on to the network. This is unlike the other two Oracle types ([Open Oracle signed messages](#open-oracle-signed-messages) & [JSON signed message data](#json-signed-message-data)),which both rely on a third party submitting data to the network.
```

Comment:
```suggestion
This will cause the network validators to read the result from the specified smart contract and submit the result to Vega. When the data is verified by enough validators, this price is accepted on to the network. This is unlike the other two Oracle types - [Open Oracle signed messages](#open-oracle-signed-messages) and [JSON signed message data](#json-signed-message-data) - which both rely on a third party submitting data to the network.
```